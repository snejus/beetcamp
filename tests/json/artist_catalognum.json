{
  "@context": "https://schema.org",
  "@id": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 3875452654
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 1
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 163716644
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 2754639698
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 3875452654
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a3875452654_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "PROCESS 404 'Cybernetic Punk Unit'"
    },
    {
      "@id": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit#b54684199",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 54684199
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "b"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 2754639698
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 257256293
        },
        {
          "@type": "PropertyValue",
          "name": "is_bfd",
          "value": true
        }
      ],
      "image": [
        "https://f4.bcbits.com/img/a0257256293_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "full digital discography (29 releases)",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "bundle_size",
            "value": 29
          },
          {
            "@type": "PropertyValue",
            "name": "discount",
            "value": 0.4
          }
        ],
        "availability": "OnlineOnly",
        "price": 91.08,
        "priceCurrency": "EUR",
        "priceSpecification": {
          "minPrice": 91.08
        },
        "url": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit#b54684199-buy"
      }
    },
    {
      "@id": "https://blvckplvgue.bandcamp.com/track/metallic-dominance",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://blvckplvgue.bandcamp.com/track/cybernetic-punk-unit",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://blvckplvgue.bandcamp.com/track/megacity-corporation",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://blvckplvgue.bandcamp.com/track/social-overdose",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@type": "MusicGroup",
    "name": "Process 404"
  },
  "copyrightNotice": "All Rights Reserved",
  "creditText": "Mastering: Axel Picodot",
  "dateModified": "18 Aug 2021 13:17:29 GMT",
  "datePublished": "12 Jun 2020 00:00:00 GMT",
  "description": "Future as always been a fantasy\r\n\r\nBlack/Plague wants to take you to the future. This distant fantasy that continues to inspire so many artists. Will technology bring us progress, or even more chaos?\r\nAntoine aka Process 404 wants you to take part. This near future where technology creates a constant conflict in our daily life. By combining elements and sonority of cyberpunk culture with Electronic, Process 404 seek to prove his vision of the future and why so many people call Techno « the sound of tomorrow». You can expect 4/4 tracks loaded with dystopian sounds and robotics ammo.",
  "image": "https://f4.bcbits.com/img/a3875452654_10.jpg",
  "keywords": [
    "Electronic",
    "acid",
    "industrial",
    "techno",
    "Paris"
  ],
  "mainEntityOfPage": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
  "name": "PROCESS 404 'Cybernetic Punk Unit'",
  "numTracks": 4,
  "publisher": {
    "@id": "https://blvckplvgue.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 2754639698
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_policies",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 1080
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 12129458
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 1080
      }
    ],
    "description": "Techno label based in Paris, France. \n\nWhen cold icy techno meets the heat torrent dancers. \n\nManaged by Ruben Hassid and Axel Picodot.",
    "foundingLocation": {
      "@type": "Place",
      "name": "Paris, France"
    },
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0012129458_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebPage",
        "name": "linktr.ee",
        "url": "https://linktr.ee/blackplaguemusic"
      },
      {
        "@type": "WebPage",
        "name": "SoundCloud",
        "url": "https://soundcloud.com/blackplaguerec"
      },
      {
        "@type": "WebPage",
        "name": "Instagram",
        "url": "https://www.instagram.com/blackplaguemusic/"
      }
    ],
    "name": "Black/Plague",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "Tracks",
        "url": "https://blvckplvgue.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "Merch",
        "url": "https://blvckplvgue.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "Us",
        "url": "https://blvckplvgue.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://blvckplvgue.bandcamp.com/track/metallic-dominance",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1255250367
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M33S",
          "mainEntityOfPage": "https://blvckplvgue.bandcamp.com/track/metallic-dominance",
          "name": "Metallic Dominance"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://blvckplvgue.bandcamp.com/track/cybernetic-punk-unit",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3964428983
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H08M34S",
          "mainEntityOfPage": "https://blvckplvgue.bandcamp.com/track/cybernetic-punk-unit",
          "name": "Cybernetic Punk Unit"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://blvckplvgue.bandcamp.com/track/megacity-corporation",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2926327708
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H07M16S",
          "mainEntityOfPage": "https://blvckplvgue.bandcamp.com/track/megacity-corporation",
          "name": "Megacity Corporation"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://blvckplvgue.bandcamp.com/track/social-overdose",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2451214870
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M23S",
          "mainEntityOfPage": "https://blvckplvgue.bandcamp.com/track/social-overdose",
          "name": "Social Overdose"
        },
        "position": 4
      }
    ],
    "numberOfItems": 4
  }
}
