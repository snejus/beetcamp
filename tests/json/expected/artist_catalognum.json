[
  {
    "album": "Cybernetic Punk Unit",
    "album_id": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "album",
    "albumtypes": [
      "album"
    ],
    "artist": "Process 404",
    "artist_credit": null,
    "artist_id": "https://blvckplvgue.bandcamp.com",
    "artist_sort": null,
    "artists": [
      "Process 404"
    ],
    "artists_credit": [],
    "artists_ids": [
      "https://blvckplvgue.bandcamp.com"
    ],
    "artists_sort": [],
    "asin": null,
    "barcode": null,
    "catalognum": "",
    "comments": "Future as always been a fantasy\n\nBlack/Plague wants to take you to the future. This distant fantasy that continues to inspire so many artists. Will technology bring us progress, or even more chaos?\nAntoine aka Process 404 wants you to take part. This near future where technology creates a constant conflict in our daily life. By combining elements and sonority of cyberpunk culture with Electronic, Process 404 seek to prove his vision of the future and why so many people call Techno « the sound of tomorrow». You can expect 4/4 tracks loaded with dystopian sounds and robotics ammo.\n---\nMastering: Axel Picodot",
    "country": "FR",
    "data_source": "bandcamp",
    "data_url": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
    "day": 12,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": "industrial, techno",
    "label": "Black/Plague",
    "language": null,
    "media": "Digital Media",
    "mediums": 1,
    "month": 6,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "release_group_title": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": "electronic",
    "tracks": [
      {
        "album": "Cybernetic Punk Unit",
        "arranger": null,
        "artist": "Process 404",
        "artist_credit": null,
        "artist_id": "https://blvckplvgue.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Process 404"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://blvckplvgue.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
        "disctitle": null,
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 393,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 4,
        "release_track_id": null,
        "title": "Metallic Dominance",
        "track_alt": null,
        "track_id": "https://blvckplvgue.bandcamp.com/track/metallic-dominance",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Cybernetic Punk Unit",
        "arranger": null,
        "artist": "Process 404",
        "artist_credit": null,
        "artist_id": "https://blvckplvgue.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Process 404"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://blvckplvgue.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
        "disctitle": null,
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 514,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 4,
        "release_track_id": null,
        "title": "Cybernetic Punk Unit",
        "track_alt": null,
        "track_id": "https://blvckplvgue.bandcamp.com/track/cybernetic-punk-unit",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Cybernetic Punk Unit",
        "arranger": null,
        "artist": "Process 404",
        "artist_credit": null,
        "artist_id": "https://blvckplvgue.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Process 404"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://blvckplvgue.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
        "disctitle": null,
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 436,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 4,
        "release_track_id": null,
        "title": "Megacity Corporation",
        "track_alt": null,
        "track_id": "https://blvckplvgue.bandcamp.com/track/megacity-corporation",
        "work": null,
        "work_disambig": null
      },
      {
        "album": "Cybernetic Punk Unit",
        "arranger": null,
        "artist": "Process 404",
        "artist_credit": null,
        "artist_id": "https://blvckplvgue.bandcamp.com",
        "artist_sort": null,
        "artists": [
          "Process 404"
        ],
        "artists_credit": [],
        "artists_ids": [
          "https://blvckplvgue.bandcamp.com"
        ],
        "artists_sort": [],
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://blvckplvgue.bandcamp.com/album/process-404-cybernetic-punk-unit",
        "disctitle": null,
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 263,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 4,
        "release_track_id": null,
        "title": "Social Overdose",
        "track_alt": null,
        "track_id": "https://blvckplvgue.bandcamp.com/track/social-overdose",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": false,
    "year": 2020
  }
]
